crates/turbopack-ecmascript-runtime/js/src/dev/runtime/base/runtime-base.ts

```diff
@@ -1116,8 +1155,8 @@ function getFirstModuleChunk(moduleId: ModuleId) {
 }
 
 /**
- * Removes a module from a chunk. Returns true there are no remaining chunks
- * including this module.
+ * Removes a module from a chunk.
+ * Returns `true` if there are no remaining chunks, including this module.
```

Comment:
This no longer means the same thing. It should be:

```
Returns `true` if there are no remaining chunks including this module.
```

Reply:
you are right, will fix in my next PR